eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -88,17 +88,17 @@ public void exec() throws IOException {
             row -> !row.exists(AssembleMojo.ATTR_EO)
                 && !row.exists(AssembleMojo.ATTR_XMIR)
         );
-        final HashOfTag tag = new HashOfTag(this.hash);
+        final ChRemote tag = new ChRemote(this.hash);
         if (this.objectionary == null) {
             this.objectionary = new OyFallbackSwap(
                 new OyHome(
-                    tag.narrow(),
+                    new ChNarrow(tag).value(),
```

Comment:
@volodya-lombrozo if you decided to switch from Strings to a new object, it's better to NOT use `.value()` so early. It's better to send `CommitHash` to `OyHome` and let `OyHome` call `.value()`. The same for `PullMojo.remote()`